app/components/ManageSpecies/index.js

```diff
@@ -64,20 +54,18 @@ const ManageSpecies = ({
         if (userSpecies && userSpecies.length > 0) {
           specieListWithUnknown = [
             ...userSpecies,
-            { guid: 'abc', isUserSpecies: true, scientific_name: 'Unknown' },
+            { guid: 'unknown', isUserSpecies: true, scientific_name: 'Unknown' },
```

Comment:
We should add a translation resource to translate the scientific name `Unknown`.

Reply:
But if we add a `i18next.t` here, this probably also has to be done at https://github.com/Plant-for-the-Planet-org/treemapper/blob/9da024767c66485739bd15f37e70752804271b2e/app/components/ManageSpecies/MySpecies.js#L60-L61 Or is it possible to change the code the way that we do not need to check against resource values?

Reply:
I used guid to check for same and added resource for Unknown, so that worked